Lex-Sample/swift-3/README.md

```diff
@@ -0,0 +1,49 @@
+# The Amazon Lex Sample
+
+This sample demonstrates how to use Amazon Lex interaction client library on iOS. This application uses AWS Cognito for authentication with Amazon Lex.
+
+## Requirements
+
+* Xcode 9.2 and later
+* iOS 9 and later
+
+## Using the Sample
+ 
+1. The AWS Mobile SDK for iOS is available through [CocoaPods](http://cocoapods.org). If you have not installed CocoaPods, install CocoaPods:
+
+		sudo gem install cocoapods
+		pod setup
+
+2. To install the AWS Mobile SDK for iOS, change the current directory to the one with your **Podfile** in it and run the following command:
+
+		pod install
+
+3. This sample requires Cognito to authorize to Amazon Lex to post content.  Use Amazon Cognito to create a new identity pool:
+    1. In the [Amazon Cognito Console](https://console.aws.amazon.com/cognito/), press the `Manage Federated Identities` button and on the resulting page press the `Create new identity pool` button.
+    1. Give your identity pool a name and ensure that `Enable access to unauthenticated identities` under the `Unauthenticated identities` section is checked.  This allows the sample application to assume the unauthenticated role associated with this identity pool.  Press the `Create Pool` button to create your identity pool.
+
+        **Important**: see note below on unauthenticated user access.
+
+    1. As part of creating the identity pool, Cognito will setup two roles in [Identity and Access Management (IAM)](https://console.aws.amazon.com/iam/home#roles).  These will be named something similar to: `Cognito_PoolNameAuth_Role` and `Cognito_PoolNameUnauth_Role`. You can view them by pressing the `View Details` button. Now press the `Allow` button to create the roles.
+	1. Save the `Identity pool ID` value that shows up in red in the "Getting started with Amazon Cognito" page, it should look similar to: `us-east-1:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" and note the region that is being used.  These will be used in the application code later.
+	1. Now we will attach a policy to the unauthenticated role which has permissions to access the required Amazon Lex API.  This is done by attaching an IAM Policy to the unauthenticated role in the [IAM Console](https://console.aws.amazon.com/iam/home#roles).  First, search for the unauth role that you created in step 3 above (named something similar to `Cognito_PoolNameUnauth_Role`) and select its hyperlink.  In the resulting "Summary" page press the `Attach Policy` button in the "Permissions" tab.
+	1. Search for "lex" and check the box next to the policy named `AmazonLexFullAccess` and then press the `Attach Policy` button.  This policy allows the application to perform all operations on the Amazon Lex service.
```

Comment:
`AmazonLexFullAccess` is a broad policy, can we scope it down?

Reply:
Yes, scoped it down to use `AmazonLexRunBotsOnly` policy instead of full access policy